landoui/app.py

```diff
@@ -73,8 +69,10 @@ def create_app(
 
     # Register routes via Flask Blueprints
     from landoui.pages import pages
```

Comment:
Yes actually! It is because these depend on the global oidc variable we set at the top.